packages/patrol_cli/lib/src/features/drive/drive_command.dart

```diff
@@ -118,6 +119,14 @@ class DriveCommand extends StagedCommand<DriveCommandConfig> {
         abbr: 'n',
         help: 'Repeat the test n times.',
         defaultsTo: '$_defaultRepeats',
+      )
+      ..addOption(
+        'use-application-binary',
+        help:
+            'Specify a pre-built application binary to use when running. For Android applications, '
+            'this must be the path to an APK. For iOS applications, the path to an IPA. Other device types '
```

Comment:
Does APP (for iOS) also work?

Reply:
I tried with `--use-application-binary=build/ios/Debug-iphonesimulator/Runner.app`, also work.

Reply:
OK 👍 

Please update the description to state this :)

Reply:
A new commit was pushed to state iOS path to a .app in help text.